s/api/types/auth.ts

```diff
@@ -1,33 +1,34 @@
-
-import * as dbmage from "dbmage"
-import {AppDatabase} from "./schema.js"
+import * as dbmage from "dbmage";
+import { AppDatabase } from "./schema.js";
 
 export interface User {
-	userId: string
-	profile: Profile
-	permissions: {
-		canPost: boolean
-		canBanUsers: boolean
-		canDeleteAnyComment: boolean
-	}
+  userId: string;
+  profile: Profile;
+  permissions: {
+    canPost?: boolean;
+    canEditOwnComment?: boolean;
```

Comment:
i don't think we need `own` permissions like `canEditOwnComment`.

if anything, this might be a setting an administrator can set as a global setting, like `usersCanEditTheirOwnComments`, but not a permission that varies from user to user.